src/style.css

```diff
@@ -0,0 +1,82 @@
+html {
+  background-color: #e5e5e5;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+}
+
+body {
+  display: table-caption;
+}
+
+.label {
+  display: inline-block;
+  position: relative;
+  margin-left: 48px;
+  background: #fff;
+  border-radius: 4px;
+}
+
+.lL {
+  /* margin-top: 71px; */
+  top: 20px;
+  width: 900px;
+  margin-bottom: 40px;
+}
+
+.lS {
+  width: 427px;
+}
+
+.img {
+  position: absolute;
+}
+
+.imgL {
```

Comment:
Don't use shortened names for your classes, also use snake-case instead of camelCase for ALL of them